OpenApi.yml

```diff
@@ -520,7 +527,6 @@ components:
     ProviderDetail:
       properties:
         fee:
-          deprecated: true
```

Comment:
please don't remove this, the lib we use to generate the OpenApi file doesn't support the deprecated field, so we need to add it manually